hoot-core-test/src/test/cpp/hoot/core/conflate/poi-polygon/PoiPolygonMatchCreatorTest.cpp

```diff
@@ -94,7 +94,7 @@ class PoiPolygonMatchCreatorTest : public HootTestFixture
 
     {
       PoiPolygonMatchCreator uut;
-      vector<const Match*> matches;
+      vector<ConstMatchPtr> matches;
```

Comment:
This is the biggest change is using `ConstMatchPtr` in all of the `matches` vectors to use shared pointers instead of raw pointers that were leaked.